tests/integration/test_keeper_ipv4_fallback/test.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pytest
+import subprocess
+from helpers.cluster import ClickHouseCluster
+
+@pytest.fixture(scope="module")
+def started_cluster(request):
+    try:
+        subprocess.check_call(["sysctl", "-w", "net.ipv6.conf.all.disable_ipv6=1"])
+    except subprocess.CalledProcessError:
+        pytest.xfail("IPv6 cannot be modified in this environment")
```

Comment:
In case the `sysctl` cannot be executed we fail on this fixture, but on the other hand we skip the test if we are not root.